troubleshooting/tutorials/remote-core-dump/README.md

```diff
@@ -196,6 +196,8 @@ Then, you can define the volume in the task definition to be your EFS filesystem
     }],
 ```
 
+As a sanity check that your setup the EFS filesystem correctly on the Fargate task, create a file in the EFS filesystem. For example, `touch my-efs-id.txt`. Then, when you later setup your Fargate task, you can use ECS Exec to check that you can see the file in the cores directory: `ls /cores`. 
```

Comment:
Grammar issue
"As a sanity check that **you** setup"

Reply:
fixed